float64 waypoint_sol_x
float64 waypoint_sag_x
float64 waypoint_sol_y
float64 waypoint_sag_y


